rfcs/text/0352-graph-type-system.md

```diff
@@ -18,17 +18,17 @@ Rather than allowing any free-form JSON schema, this RFC defines a set of JSON s
 
 These elements are used in combination to improve reusability of, and better define, expressions of data structural requirements.
 
-By using the proposed types, blocks, embedding applications, and end-users can all have greater confidence in each other's roles and functionalities, and hopefully will allow them all to benefit from better computational inference and capabilities. Reusability of descriptions of data should encourage a decrease in developmental overhead overtime, with all members of the ecosystem benefiting from emergent standardization.
+By using the proposed types, blocks, embedding applications, and end-users can all have greater confidence in each other's roles and functionalities, and hopefully will allow them all to benefit from better computational inference and capabilities. Reusability of descriptions of data should decrease developmental overhead, with all members of the ecosystem benefiting from emergent standardization.
```

Comment:
I appreciate this was quite wordy but

> should encourage a decrease in developmental overhead over time

Is more correct. The sentence was highlighting that developmental overhead will decrease _over time_. For a long time it will actually increase, and this is a risk we have to be aware of. It's only when we reach some form of critical mass that people will start to largely benefit from it.